artman/cli/main.py

```diff
@@ -11,34 +11,41 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""The new artman CLI with the following syntax.
 
-"""CLI to execute pipeline either locally or remotely."""
+    artman [Options] generate|publish <artifact_name>
+
+.. note::
+    Only local execution is supported at this moment. The CLI syntax is
+    beta, and might have changes in the future.
+"""
 
 from __future__ import absolute_import
-from logging import DEBUG, INFO
+from logging import INFO
 import argparse
-import ast
-import base64
+from distutils.dir_util import copy_tree
 import io
 import os
+import pprint
+import subprocess
 import sys
-import tempfile
-import time
-import uuid
-
-from gcloud import logging
 
 from ruamel import yaml
-
 from taskflow import engines
 
+from artman.config import converter, loader
+from artman.config.proto.config_pb2 import Artifact, Config
 from artman.cli import support
 from artman.pipelines import pipeline_factory
-from artman.utils import job_util, pipeline_util, config_util
+from artman.utils import config_util
 from artman.utils.logger import logger, setup_logging
 
+ARTMAN_DOCKER_IMAGE = 'googleapis/artman:0.5.0'
```

Comment:
(doesn't have to be addressed in this PR): will this ever be if a format other than `googleapis/artman:{latest-version}`?

if not, consider making the version a global constant and referencing it both from here and `setup.py` to avoid having to update it in two places.

Reply:
I don't have a plan to change the format. It makes sense to make version a constant. I didn't realize that setup.py is a python file itself.